### PR TITLE
{bio}[system/system] mgltools v1.5.7

### PR DESCRIPTION
**Pull request by guacke**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/88c972bf1beae72c4b3d7117211d5346 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3bc9b33132ae71f0469aa0b59fe53048 for a full test report.

File: easybuild/easyconfigs/m/mgltools/mgltools-1.5.7.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+name = "mgltools"
+version = "1.5.7"
+
+homepage = "https://ccsb.scripps.edu/mgltools/"
+description = """
+   The MGLTools software suite can be used for visualization 
+   and analysis of molecular structures and comprises 
+   the Python Molecular Viewer (PMV, a general purpose molecular viewer), 
+   AutoDockTools (ADT, a set of PMV commands specifically developed to support AutoDock users) 
+   and Vision (a visual programing environment).
```
Comment (verdurin) [inline easybuild/easyconfigs/m/mgltools/mgltools-1.5.7.eb:12]:
```suggestion
   and Vision (a visual programming environment).
```

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3106.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/fafa3bf2378c465406b23e936deb8766 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @guacke!

**Pull request merged**